src/lib/settings/cdk/settings_validator.py

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
when I try to validate settings in app.py (code below) :

FileNotFoundError: File not found: src/lib/settings/cdk/schemas/general.json

# settings_provider = LocalFileSystemSettingsProvider("../config/settings")
# settings_provider.validate_settings()

current_settings = Settings.from_file_path("../config/settings")
settings_validator.validate(current_settings)

Reply:
fixed: added full path as discussed :)